CMakeLists.txt

```diff
@@ -213,6 +213,19 @@ else()
   set(LIBTYPE SHARED)
 endif()
 
+
+foreach(rpath IN ITEMS
```

Comment:
How it affects conda build?

Reply:
It shouldn't. Conda build doesn't care about those paths anyway.

Reply:
OK, we are overwriting them anyway https://github.com/NVIDIA/DALI/blob/main/conda/dali_native_libs/recipe/build.sh#L216.

Reply:
Yes